src/modules/sub-items/sub.items.module.js

```diff
@@ -247,7 +243,7 @@ export default class SubItemsModule extends Component {
      * @memberof SubItemsModule
      */
     switchView(activeView) {
-        this.setState(() => ({ activeView }));
+        this.setState(() => ({ activeView }), eZ.helpers.tooltips.hideAll());
```

Comment:
The second parameter in `setState` is a function. I don't know how this will work.
```suggestion
        this.setState(() => ({ activeView }), () => eZ.helpers.tooltips.hideAll());
```